pipeline-adapters/mlrun-pipelines-kfp-v1-8/setup.py

```diff
@@ -21,7 +21,7 @@
 
 setup(
     name="mlrun-pipelines-kfp-v1-8",
-    version="0.2.3",
+    version="0.2.4",
```

Comment:
because this is a breaking change (even between rcs) we would need to bump its minor to ensure previous rcs can still run and get "kubernetes" as expected


```
  File "/conda/lib/python3.9/site-packages/mlrun/config.py", line 890, in update
    if hasattr(self, key):
  File "/conda/lib/python3.9/site-packages/mlrun/config.py", line 1199, in version
    from mlrun.utils.version import Version
  File "/conda/lib/python3.9/site-packages/mlrun/utils/__init__.py", line 15, in <module>
    from .async_http import AsyncClientWithRetry  # noqa
  File "/conda/lib/python3.9/site-packages/mlrun/utils/async_http.py", line 29, in <module>
    from .helpers import logger as mlrun_logger
  File "/conda/lib/python3.9/site-packages/mlrun/utils/helpers.py", line 54, in <module>
    from mlrun_pipelines.models import PipelineRun
  File "/conda/lib/python3.9/site-packages/mlrun_pipelines/models.py", line 23, in <module>
    from mlrun_pipelines.imports import ContainerOp
  File "/conda/lib/python3.9/site-packages/mlrun_pipelines/imports.py", line 19, in <module>
    from kubernetes.client import V1VolumeMount
ModuleNotFoundError: No module named 'kubernetes' 
```

when it does not get the kfp